<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxDebugState">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="title">
         <string>调试状态</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxSerial">
           <property name="text">
            <string>串口已连接</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBoxNetwork">
           <property name="text">
            <string>网络已连接</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnCancelSerialPortCon">
           <property name="text">
            <string>取消串口连接</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnCancelNetCon">
           <property name="text">
            <string>取消网络连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxSensorDynaZMP">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>传感器+舵机+ZMP(sdz文件)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEditFilePathSdz">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnfilePathSensorDynaZMP">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>选择存储文件路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnStartGetData">
             <property name="text">
              <string>开始获取数据</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStopGetData">
             <property name="text">
              <string>停止获取数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBoxGaitCorrect">
      <property name="title">
       <string>计算矫正步态</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLineEdit" name="lineEditSelSdzFilePath">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSelSdzFilePath">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>选择sdz文件...</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>方法1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>方法2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>方法3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>方法4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>方法5</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>方法6</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBoxGaitData">
      <property name="title">
       <string>步态数据</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="plainTextEditGaitData">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonClearText">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="serialPort"/>
    <addaction name="networkCom"/>
    <addaction name="sshConnect"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpDoc"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>可视化</string>
    </property>
    <addaction name="nineAxisSensor"/>
    <addaction name="gaitCorrect"/>
    <addaction name="steeringEngine"/>
    <addaction name="zmpTrajectory"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFile"/>
   <addaction name="saveFile"/>
   <addaction name="separator"/>
   <addaction name="serialPort"/>
   <addaction name="networkCom"/>
   <addaction name="sshConnect"/>
   <addaction name="separator"/>
   <addaction name="exit"/>
  </widget>
  <action name="serialPort">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/export1.bmp</normaloff>:/images/images/export1.bmp</iconset>
   </property>
   <property name="text">
    <string>连接串口</string>
   </property>
   <property name="toolTip">
    <string>连接串口</string>
   </property>
  </action>
  <action name="networkCom">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/066.GIF</normaloff>:/images/images/066.GIF</iconset>
   </property>
   <property name="text">
    <string>连接网络</string>
   </property>
   <property name="toolTip">
    <string>连接网络</string>
   </property>
  </action>
  <action name="helpDoc">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/135.JPG</normaloff>:/images/images/135.JPG</iconset>
   </property>
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="sshConnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/090.GIF</normaloff>:/images/images/090.GIF</iconset>
   </property>
   <property name="text">
    <string>SSH远程连接</string>
   </property>
  </action>
  <action name="nineAxisSensor">
   <property name="text">
    <string>九轴传感器</string>
   </property>
  </action>
  <action name="gaitCorrect">
   <property name="text">
    <string>矫正步态</string>
   </property>
   <property name="toolTip">
    <string>矫正步态</string>
   </property>
  </action>
  <action name="steeringEngine">
   <property name="text">
    <string>舵机</string>
   </property>
  </action>
  <action name="zmpTrajectory">
   <property name="text">
    <string>ZMP轨迹</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="newFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="openFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/save.bmp</normaloff>:/images/images/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="anotherSaveFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
